src/server.jl

```diff
@@ -80,9 +81,85 @@ function to_param(T, source::Dict, name::String; required::Bool=false, collectio
     end
 end
 
+function is_any_mime_accepted(mime_type::AbstractString)
+    parts = first(strip.(split(mime_type, ";")))
+    if parts == "*/*"
+        return true
+    end
+    return false
+end
+
+# json if it is of the format `.*/json[;.*]` (roughly)
+function is_json_mime_accepted(mime_type::AbstractString)
+    parts = first(strip.(split(mime_type, ";")))
+    parts = strip.(split(parts, "/"))
+    return (length(parts) == 2) && (parts[2] == "json")
```

Comment:
Should we just regex match on the `split` string instead (same below)?

Reply:
Did consider regex.. but `split` seemed simpler/easier to understand to me.